make sure to set platform=NULL in wiringXGC()

Artox opened the pull request:
In case somebody gets the idea to run wiringXGC() multiple times,
the call to platform->gc() will happen on free'd memory!

CurlyMoo commented:
Can you commit under Artox?

Artox commented:
Oh committer name
Hmmmmm is that important?

CurlyMoo commented:
It makes it easier to contact the committer afterwards on github?

Artox commented:
does it? I am not sure, but I can change it no problem

Artox commented:
done

Artox commented:
also I made a little fun project of mine: Java Bindings. Feel free to have a look but it will be quite a while before they are finished.
Current state is: all GPIO functions implemented.
https://github.com/Artox/wiringX-java

CurlyMoo commented:
I think this email address is not part of your github account:
https://api.github.com/repos/mxOBS/wiringX/commits/18c10f3962dd39466b07447d12f2c89eaa1611a9

Artox commented:
Indeed it wasn't until about an hour ago when I noticed.
This happened because I switched from fake email privacy@not.given to valid email

Artox commented:
I guess I can change name back then?

Artox committed:
properly clean up state in wiringXGC()

In case somebody gets the idea to run wiringXGC() multiple times,
the call to platform->gc() will happen on free'd memory!
Also unset issetup so any later call to wiringXSetup can succeed.

Artox committed:
return after detecting platform=NULL (dont crash on purpose)
